docs/content/4.api/4.advanced.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 'Advanced'
+description: 'Nuxt Content is highly customizable, giving you freedom and control over how the data is transformed.'
+icon: heroicons-outline:lightning-bolt
+---
+
+## Hooks
+
+The module adds some hooks you can use:
+
+::alert
+In order to use `content:file:*` hooks you need to create custom [nitro plugin](https://nitro.unjs.io/guide/plugins.html).
+::
+
```

Comment:
Would be nice to add a section like:

```ts
export default defineNuxtConfig({
  // ...
  nitro: {
    plugins: ['~/server/content.ts']
  }
})
```

Hopefully we won't have to do it once https://github.com/nuxt/framework/issues/5211 will be available.